src/read-persons.ts

```diff
@@ -1,16 +1,6 @@
 import { errorWithCause } from "@sap-cloud-sdk/util";
 import { Person } from "./model/person";
 
-export async function personById(srv, personId: string): Promise<Person> {
-  return handleError(srv.read("Person", { ID: personId }))
-    .then((persons: Person[]) => {
-      if (persons.length === 0) {
-        throw new Error(`No person found with ID ${personId}`);
-      }
-      return persons[0];
-    });
-}
-
 export async function readPersons(srv): Promise<Person[]> {
```

Comment:
Just a general question - is this still used?

Reply:
yes read-appointments line 15

Reply:
When you delete this but to not change `read-appointments`, `read-appointments` is not broken?

Reply:
if I delete readPersons, read-appointments is broken